test-cypress/integration/explore/similarity.spec.js

```diff
@@ -464,4 +582,12 @@ describe('Similarity screen validation', () => {
         const query = 'OPTIONAL { ?result <http://dbpedia.org/ontology/birthPlace> ?birthDate .';
         YasqeSteps.verifyQueryContains(query);
     }
+
+    function createVerifyConfirmationDialogOptions() {
```

Comment:
I have moved all such functions to respective steps in graphs config tests I'm reworking now